Tchap/Config/BuildSettings.swift

```diff
@@ -251,6 +251,8 @@ final class BuildSettings: NSObject {
     
     static let allowInviteExernalUsers: Bool = true
     
+    static let allowBackgroundAudioMessagePlayback: Bool = true
```

Comment:
Please add this change in Btchap and DevTchap `BuildSettings.swift` files too.

Reply:
Done